broker-daemon/broker-rpc/wallet-service/release-channels.js

```diff
@@ -84,6 +84,12 @@ async function releaseChannels ({ params, logger, engines, orderbooks }, { Relea
     closeChannels(counterEngine, counterSymbol, force, logger)
   ])
 
+  if (base.status === RELEASE_STATE.RELEASED && counter.status === RELEASE_STATE.RELEASED) {
+    const { successfulCancellations, unsuccessfulCancellations } = await blockOrderWorker.cancelActiveOrders(market)
```

Comment:
Probably want this to happen before, or in tandem with the channels closing. One of the channels failing to close resulting in the orders not being cancelled is a bit of weird behavior I think.